oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/parser/MetricsHolder.java

```diff
@@ -45,13 +48,16 @@ public static void init() throws IOException {
         }
     }
 
-    public static Class<? extends Metrics> find(
-        String functionName) {
-        String func = functionName;
-        Class<? extends Metrics> metricsClass = REGISTER.get(
-            func);
+    @SneakyThrows
```

Comment:
AFAIK, IllegalArgumentException is a RuntimeException. Why need this?

Reply:
`init` throws `IOException`, which is not runtime exception